test/tests/functional/pbs_qrun.py

```diff
@@ -95,52 +95,51 @@ def test_invalid_host_val(self):
 
     def test_qrun_hangs(self):
         """
-        This test verifies whether qrun hangs, if scheduling is
-        disabled
+        This test submit 500 jobs with differnt equivalence class,
+        turn of scheduling and qrun job to
+        verify whether qrun hangs.
         """
         node = self.mom.shortname
-        time1 = int(time.time())
-        walltime = 1
+        err_msg = 'qrun: Not enough free nodes available'
+        joblist = []
         self.server.manager(MGR_CMD_SET, SCHED,
                             {'scheduling': 'False'})
-        err_msg = 'qrun: Not enough free nodes available'
-        while True:
-            self.server.manager(MGR_CMD_SET, NODE,
-                                {'resources_available.ncpus': 1}, id=node)
-
-            j1 = Job(TEST_USER)
-            j1.create_script('/bin/date')
-            a = {'Resource_List.walltime': walltime}
-            j1.set_attributes(a)
-            self.server.submit(j1)
-
-            walltime = walltime + 1
+        self.server.manager(MGR_CMD_SET, NODE,
+                            {'resources_available.ncpus': 1}, id=node)
+        for walltime in range(1, 501):
+            j = Job(TEST_USER)
+            j.create_script('/bin/date')
```

Comment:
You aren't really going to run these jobs, so the default sleep job should be fine.

Reply:
Sure will keep it default. Thanks